src/annotator/annotation-counts.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var events = require('../sidebar/events');
```

Comment:
I see a couple of problems here:

1. Part of the purpose of separating the sidebar and annotator code into separate directories is to make it enforce isolation of the two different parts of the client and prevent inappropriate dependencies. Specifically:

* Modules in `annotator/` and `sidebar/` can only depend on code in the same module or in `shared/`
* Modules in `shared/` can only depend on other modules in `shared/`

2. Mixing up constants for events on different buses (Angular scopes, events across the bridge) in `sidebar/events` is a source of confusion. How about creating a separate file (eg. `shared/bridge-events`) which enumerates and documents the events sent across the bridge between the sidebar and annotator?